translations/README.cs.md

```diff
@@ -46,38 +46,42 @@ cd React-GitHub-Resume
 
 2. Nainstaluj balíčky
 
-    Používáš-li `npm`
-    ```bash
-    npm install
-    ```
+   Používáš-li `npm`
 
-    nebo
+   ```bash
+   npm install
+   ```
 
-    používáš-li `yarn`
-    ```bash
-    yarn
-    ```
+   nebo
+
+   používáš-li `yarn`
+
+   ```bash
+   yarn
+   ```
 
 3. Spusť vývojářský server
 
-    Používáš-li `npm`
-    ```bash
-    npm start
-    ```
+   Používáš-li `npm`
+
+   ```bash
+   npm start
+   ```
+
+   nebo
 
-    nebo
+   používáš-li `yarn`
 
-    používáš-li `yarn`
-    ```bash
-    yarn start
-    ```
+   ```bash
+   yarn start
+   ```
 
 4. Přejdi na adresu <http://localhost:3000>
 
 ## [Živé demo](https://react-github-resume.vercel.app/)
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
 
-## 📄 Licence
+## 📄 licenci
 
 Tento projekt spadá pod licenci MIT — pro více detailů navštiv soubor s [LICENCÍ](./LICENSE)

```

Comment:
Please, correct the `LICENSE` route to `../LICENSE`

Reply:
fixed